vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -170,19 +170,25 @@ class MessageActionsViewModel @AssistedInject constructor(@Assisted
     private fun computeMessageBody(timelineEvent: TimelineEvent): CharSequence {
         return when (timelineEvent.root.getClearType()) {
             EventType.MESSAGE,
+            EventType.ENCRYPTED,
```

Comment:
I think it's strange to add this line, because in the `when` we test the clear type (`getClearType()`).
Maybe handle `timelineEvent.root.isRedacted()` before the `when` statement is better because it will also manage redaction of state events

Reply:
I will, but we may investigate that later; why clearType of redacted event is determined as ENCRYPTED.

Reply:
ok. Please do not resolve the conversation, or I do not see your comment :)